Publication.create!(authors: "Konikoff, J., and Toplosky, S.", date: "2010", title: "Analysis of Simplified DES Algorithms", journal: "Cryptologia", issue: "34(3)", pages: "211-224", external_link: "http://dx.doi.org/10.1080/01611191003646433", abstract: "The Data Encryption Standard (DES) is no longer secure because of its small key size. One way to augment the keyspace is to perform multiple encryptions with DES. This paper examines the cryptographic consequences of multiple encryptions with DES through two simplified DES algorithms. Both simplified algorithms, under multiple encryptions, generate the alternating group of permutations on the message space. Moreover, equivalent keys and key pairs are rare in both cases. These facts support past work that has shown that performing multiple encryptions with DES adds keyspace by demonstrating that the increase in keyspace is significant.")

Publication.create!(authors: "Brookmeyer, R., and Konikoff, J.", date: "2011", title: "Statistical Considerations in Determining HIV Incidence from Changes in HIV Prevalence", journal: "Statistical Communications in Infectious Diseases", issue: "3(1)", pages: "article 9", external_link: "https://works.bepress.com/rbrookmeyer/34/", abstract: "The development of methods for estimating HIV incidence is critical for tracking the epidemic and for designing, targeting and evaluating HIV prevention efforts. One method for estimating incidence is based on changes in HIV prevalence. That method is attracting increased attention because national population-based HIV prevalence surveys, such as Demographic and Health Surveys, are being conducted throughout the world. Here, we consider some statistical issues associated with estimating HIV incidence from two population-based HIV prevalence surveys conducted at two different points in time. We show that the incidence estimator depends on the relative survival rate. We evaluate the sensitivity of estimates to incorrect assumptions about the relative survival rate, and show that small errors in the relative survival can, in some situations, create large biases in HIV incidence. We determine sample sizes of prevalence surveys to estimate incidence with precision and show how the sample sizes depend on baseline prevalence, the relative survival rate, and the population HIV incidence rate. We find that even if the relative survival rate were known exactly, there are situations where prohibitively large prevalence surveys would be required to produce reliable incidence estimates. These situations can occur either when the baseline prevalence is large, the relative survival rate is near 1, or the population incidence is small. Because information on the relative survival rate may be limited or not specific to the population under study, we suggest an approach to empirically estimate this critical parameter by augmenting population-based prevalence surveys with a mortality follow-up sub-study. We determine sample sizes of the prevalence surveys and mortality sub-studies for this augmented design and provide the necessary R code (version 2.13.0) for sample size determinations. We conclude that caution should be exercised when solely relying on changes in prevalence as the method for determining HIV incidence because of the method's sensitivity to mortality assumptions and the very large sample size requirements in some settings.")

Publication.create!(authors: "Laeyendecker, O., Brookmeyer, R., Cousins, M. M., Mullis, C. E., Konikoff, J., Donnell, D., Celum, C., Buchbinder, S. P., Seage, G. R. III, Kirk, G. D., Mehta S. H., Astemborksi, J., Jacobson, L. P., Margolick, J. B., Brown J., Quinn, T. C., and Eshleman S. H.", date: "2013", title: "HIV Incidence Determination in the United States: A Multiassay Approach", journal: "Journal of Infectious Diseases", issue: "207", pages: "232-239", external_link: "https://academic.oup.com/jid/article/207/2/232/2192406/HIV-Incidence-Determination-in-the-United-States-A", background: "Accurate testing algorithms are needed for estimating human immunodeficiency virus (HIV) incidence from cross-sectional surveys.", methods: "We developed a multiassay algorithm (MAA) for HIV incidence that includes the BED capture enzyme immunoassay (BED-CEIA), an antibody avidity assay, HIV load, and CD4+ T-cell count. We analyzed 1782 samples from 709 individuals in the United States who had a known duration of HIV infection (range, 0 to >8 years). Logistic regression with cubic splines was used to compare the performance of the MAA to the BEDCEIA and to determine the window period of the MAA. We compared the annual incidence estimated with the MAA to the annual incidence based on HIV seroconversion in a longitudinal cohort.", results: "The MAA had a window period of 141 days (95% confidence interval [CI], 94–150) and a very low false-recent misclassification rate (only 0.4% of 1474 samples from subjects infected for >1 year were misclassified as indicative of recent infection). In a cohort study, annual incidence based on HIV seroconversion was 1.04% (95% CI, .70%–1.55%). The incidence estimate obtained using the MAA was essentially identical: 0.97% (95% CI, .51%–1.71%).", conclusions: "The MAA is as sensitive for detecting recent HIV infection as the BED-CEIA and has a very low rate of false-recent misclassification. It provides a powerful tool for cross-sectional HIV incidence determination.")

Publication.create!(authors: "Brookmeyer, R., Konikoff J., Laeyendecker, O., and Eshleman, S.H.", date: "2013", title: "Estimation of HIV Incidence Using Multiple Biomarkers", journal: "American Journal of Epidemiology", issue: "49(10)", pages: "264-272", external_link: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3626051/", abstract: "The incidence of human immunodeficiency virus (HIV) is the rate at which new HIV infections occur in populations. The development of accurate, practical, and cost-effective approaches to estimation of HIV incidence is a priority among researchers in HIV surveillance because of limitations with existing methods. In this paper, we develop methods for estimating HIV incidence rates using multiple biomarkers in biological samples collected from a cross-sectional survey. An advantage of the method is that it does not require longitudinal follow-up of individuals. We use assays for BED, avidity, viral load, and CD4 cell count data from clade B samples collected in several US epidemiologic cohorts between 1987 and 2010. Considering issues of accuracy, cost, and implementation, we identify optimal multiassay algorithms for estimating incidence. We find that the multiple-biomarker approach to cross-sectional HIV incidence estimation corrects the significant deficiencies of currently available approaches and is a potentially powerful and practical tool for HIV surveillance.")

Publication.create!(authors: "Konikoff, J., Brookmeyer, R., Longosz, A. F., Cousins, M. M., Celum, C., Buchbinder, S. P., Seage, G. R. III, Kirk, G. D., Moore, R. D., Mehta S. H., Margolick, J. B., Brown J., Mayer, K. H., Koblin, B. A., Justman, J. E., Hodder, S. L., Quinn, T. C., Eshleman S. H., and Laeyendecker, O.", date: "2013", title: "Performance of a Limiting-Antigen Avidity Enzyme Immunoassay for Cross-Sectional Estimation of HIV Incidence in the United States", journal: "PLoS ONE", issue: "8(12)", pages: "e82772", external_link: "http://dx.doi.org/10.1371/journal.pone.0082772", background: "A limiting antigen avidity enzyme immunoassay (HIV-1 LAg-Avidity assay) was recently developed for cross-sectional HIV incidence estimation. We evaluated the performance of the LAg-Avidity assay alone and in multi-assay algorithms (MAAs) that included other biomarkers.", methods_and_findings: "Performance of testing algorithms was evaluated using 2,282 samples from individuals in the United States collected 1 month to >8 years after HIV seroconversion. The capacity of selected testing algorithms to accurately estimate incidence was evaluated in three longitudinal cohorts. When used in a single-assay format, the LAg-Avidity assay classified some individuals infected >5 years as assay positive and failed to provide reliable incidence estimates in cohorts that included individuals with long-term infections. We evaluated >500,000 testing algorithms, that included the LAg-Avidity assay alone and MAAs with other biomarkers (BED capture immunoassay [BED-CEIA], BioRad-Avidity assay, HIV viral load, CD4 cell count), varying the assays and assay cutoffs. We identified an optimized 2-assay MAA that included the LAg-Avidity and BioRad-Avidity assays, and an optimized 4-assay MAA that included those assays, as well as HIV viral load and CD4 cell count. The two optimized MAAs classified all 845 samples from individuals infected >5 years as MAA negative and estimated incidence within a year of sample collection. These two MAAs produced incidence estimates that were consistent with those from longitudinal follow-up of cohorts. A comparison of the laboratory assay costs of the MAAs was also performed, and we found that the costs associated with the optimal two assay MAA were substantially less than with the four assay MAA.", conclusions: "The LAg-Avidity assay did not perform well in a single-assay format, regardless of the assay cutoff. MAAs that include the LAg-Avidity and BioRad-Avidity assays, with or without viral load and CD4 cell count, provide accurate incidence estimates.")

Publication.create!(authors: "Cousins, M. M., Konikoff, J., Laeyendecker, O., Celum, C., Buchbinder, S. P., Seage, G. R. III, Kirk, G. D., Moore, R. D., Mehta S. H., Margolick, J. B., Brown J., Mayer, K. H., Koblin, B. A., Wheeler, D., Justman, J. E., Hodder, S. L., Quinn, T. C., Brookmeyer, R., and Eshleman S. H.", date: "2014", title: "HIV Diversity as a Biomarker for HIV Incidence Estimation: Including a High-Resolution Melting Diversity Assay in a Multiassay Algorithm", journal: "Journal of Clinical Microbiology", issue: "52(1)", pages: "115-121", external_link: "http://jcm.asm.org/content/52/1/115.full", abstract: "Multiassay algorithms (MAAs) can be used to estimate cross-sectional HIV incidence. We previously identified a robust MAA that includes the BED capture enzyme immunoassay (BED-CEIA), the Bio-Rad Avidity assay, viral load, and CD4 cell count. In this report, we evaluated MAAs that include a high-resolution melting (HRM) diversity assay that does not require sequencing. HRM scores were determined for eight regions of the HIV genome (2 in gag, 1 in pol, and 5 in env). The MAAs that were evaluated included the BED-CEIA, the Bio-Rad Avidity assay, viral load, and the HRM diversity assay, using HRM scores from different regions and a range of region-specific HRM diversity assay cutoffs. The performance characteristics based on the proportion of samples that were classified as MAA positive by duration of infection were determined for each MAA, including the mean window period. The cross-sectional incidence estimates obtained using optimized MAAs were compared to longitudinal incidence estimates for three cohorts in the United States. The performance of the HRM-based MAA was nearly identical to that of the MAA that included CD4 cell count. The HRM-based MAA had a mean window period of 154 days and provided cross-sectional incidence estimates that were similar to those based on cohort follow-up. HIV diversity is a useful biomarker for estimating HIV incidence. MAAs that include the HRM diversity assay can provide accurate HIV incidence estimates using stored blood plasma or serum samples without a requirement for CD4 cell count data.")

Publication.create!(authors: "Cousins, M. M., Konikoff, J., Sabin, D., Khaki, L., Longosz, A. F., Laeyendecker, O., Celum, C., Buchbinder, S. P., Seage, G. R. III, Kirk, G. D., Moore, R. D., Mehta S. H., Margolick, J. B., Brown J., Mayer, K. H., Koblin, B. A., Wheeler, D., Justman, J. E., Hodder, S. L., Quinn, T. C., Brookmeyer, R., and Eshleman S. H.", date: "2014", title: "A Comparison of Two Measures of HIV Diversity in Multi-Assay Algorithms for HIV Incidence Estimation", journal: "PLoS ONE", issue: "9(6)", pages: "e101043", external_link: "http://dx.doi.org/10.1371/journal.pone.0101043", background: "Multi-assay algorithms (MAAs) can be used to estimate HIV incidence in cross-sectional surveys. We compared the performance of two MAAs that use HIV diversity as one of four biomarkers for analysis of HIV incidence.", methods: "Both MAAs included two serologic assays (LAg-Avidity assay and BioRad-Avidity assay), HIV viral load, and an HIV diversity assay. HIV diversity was quantified using either a high resolution melting (HRM) diversity assay that does not require HIV sequencing (HRM score for a 239 base pair env region) or sequence ambiguity (the percentage of ambiguous bases in a 1,302 base pair pol region). Samples were classified as MAA positive (likely from individuals with recent HIV infection) if they met the criteria for all of the assays in the MAA. The following performance characteristics were assessed: (1) the proportion of samples classified as MAA positive as a function of duration of infection, (2) the mean window period, (3) the shadow (the time period before sample collection that is being assessed by the MAA), and (4) the accuracy of cross-sectional incidence estimates for three cohort studies.", results: "The proportion of samples classified as MAA positive as a function of duration of infection was nearly identical for the two MAAs. The mean window period was 141 days for the HRM-based MAA and 131 days for the sequence ambiguity-based MAA. The shadows for both MAAs were <1 year. Both MAAs provided cross-sectional HIV incidence estimates that were very similar to longitudinal incidence estimates based on HIV seroconversion.", conclusions: "MAAs that include the LAg-Avidity assay, the BioRad-Avidity assay, HIV viral load, and HIV diversity can provide accurate HIV incidence estimates. Sequence ambiguity measures obtained using a commercially-available HIV genotyping system can be used as an alternative to HRM scores in MAAs for cross-sectional HIV incidence estimation.")

Publication.create!(authors: "Konikoff, J. and Brookmeyer R.", date: "2015", title: "Sample Size Methods for Estimating HIV Incidence from Cross-Sectional Surveys", journal: "Biometrics", issue: "71(4)", pages: "1121-1129", external_link: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4715554/", abstract: "Understanding HIV incidence, the rate at which new infections occur in populations, is critical for tracking and surveillance of the epidemic. In this paper we derive methods for determining sample sizes for cross-sectional surveys to estimate incidence with sufficient precision. We further show how to specify sample sizes for two successive cross-sectional surveys to detect changes in incidence with adequate power. In these surveys biomarkers such as CD4 cell count, viral load, and recently developed serological assays are used to determine which individuals are in an early disease stage of infection. The total number of individuals in this stage, divided by the number of people who are uninfected, is used to approximate the incidence rate. Our methods account for uncertainty in the durations of time spent in the biomarker defined early disease stage. We find that failure to account for this uncertainty when designing surveys can lead to imprecise estimates of incidence and underpowered studies. We evaluated our sample size methods in simulations and found that they performed well in a variety of underlying epidemics. Code for implementing our methods in R is available with this paper at the Biometrics website on Wiley Online Library.")
